Projects/App/Sources/AppDelegate.swift

```diff
@@ -201,11 +200,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             return InterceptingURLSessionClient()
         }
         setupAnalyticsAndTracking()
-        bag += Localization.Locale.$currentLocale
-            .onValue { locale in
+
+        Localization.Locale.$currentLocale
+            .plain()
+            .publisher
+            .receive(on: RunLoop.main)
+            .sink { locale in
                 ApplicationState.setPreferredLocale(locale)
                 ApolloClient.acceptLanguageHeader = locale.acceptLanguageHeader
             }
+            .store(in: &cancellables)
```

Comment:
Its questionable when do we remove all cancellables from the array.

If for example we call this function 'setupSession' multiple times it will observe this property and execute the same things multiple times
```
                ApplicationState.setPreferredLocale(locale)
                ApolloClient.acceptLanguageHeader = locale.acceptLanguageHeader
```
Its either should be always called once and just once or have a separated AnyCancellable that would cancel previous one.

So moving this 
```
        Localization.Locale.$currentLocale
            .plain()
            .publisher
            .receive(on: RunLoop.main)
            .sink { locale in
                ApplicationState.setPreferredLocale(locale)
                ApolloClient.acceptLanguageHeader = locale.acceptLanguageHeader
            }
            .store(in: &cancellables)
```
to 
```
 func application(
        _: UIApplication,
        didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?
    ) -> Bool {
```
Right after setupSession()

Or create 
```
class AppDelegate: UIResponder, UIApplicationDelegate {
    var cancellables = Set<AnyCancellable>()
    private var localeCancellable: AnyCancellable? //this one
```